### PR TITLE
Add navigation to build page

### PR DESCRIPTION
**Pull request by MilenaHristova**

### CODE REVIEW

File: src/ProductConstructionService/ProductConstructionService.BarViz/Pages/Build.razor [hunk 1]
```diff
@@ -15,21 +15,38 @@
...
 
-    protected override async Task OnInitializedAsync()
+    protected override async Task OnParametersSetAsync()
     {
-        _build = (await PcsApi.Builds.GetLatestAsync(repository: _repo, channelId: _channelId));
+        Repo = Uri.UnescapeDataString(RepoEscaped!);
+
+        if (BuildId == "latest")
+        {
+            _build = (await PcsApi.Builds.GetLatestAsync(repository: Repo, channelId: ChannelId));
```
Comment (premun) [inline src/ProductConstructionService/ProductConstructionService.BarViz/Pages/Build.razor:45]:
```suggestion
            _build = await PcsApi.Builds.GetLatestAsync(repository: Repo, channelId: ChannelId);
```

**Review by oleksandr-didyk — APPROVED** [review]

File: src/ProductConstructionService/ProductConstructionService.BarViz/Pages/Build.razor [hunk 2]
```diff
@@ -15,21 +15,38 @@
...
+        }
+        else
+        {
+            _build = (await PcsApi.Builds.GetBuildAsync(int.Parse(BuildId!)));
```
Comment (premun) [inline src/ProductConstructionService/ProductConstructionService.BarViz/Pages/Build.razor:49]:
```suggestion
            _build = await PcsApi.Builds.GetBuildAsync(int.Parse(BuildId!));
```

File: src/ProductConstructionService/ProductConstructionService.BarViz/Pages/Build.razor [hunk 3]
```diff
@@ -1,4 +1,4 @@
-﻿@page "/build"
+﻿@page "/channel-{ChannelId:int}/{RepoEscaped}/build-{BuildId}"
```
[the remark this answers is not in the document]
Comment (MilenaHristova) [reply src/ProductConstructionService/ProductConstructionService.BarViz/Pages/Build.razor:1]: I agree

[the remark this answers is not in the document]
Comment (MilenaHristova) [reply src/ProductConstructionService/ProductConstructionService.BarViz/Pages/Build.razor:1]: @pavel-purma ?

**Review by premun — APPROVED** [review]
Can you make sure it works with the change from Pavel too?

**Review by premun — APPROVED** [review]

**Pull request merged**